### ISSUE TITLE
Tooltips should be exposed even if the control is disabled

### ISSUE THREAD

**Issue by jgrieves** [microsoft/microsoft-ui-xaml#1149]
**Describe the bug**
We have buttons that have tooltips.  The buttons are disabled because they cannot be active given the state of the UI, but users still want to know what the buttons are for (and what the keyboard shortcuts are 

**Steps to reproduce the bug**
1.  Create a XAML app with disabled buttons that have tooltips
2. Hover over the button with the mouse

**Expected behavior**
Tooltip appears

**Actual behavior**
No tooltip

**Additional context**
During a Microsoft User study this frustrated 4 of 5 participants.  The recommendation from the UX researchers was to always show tooltips.

**Comment by jevansaks** [comment]
I think this would require disabled controls to get mouse input but it's something we've been asked to do in the past.

**Comment by YuliKl** [comment]
@jgrieves, the issue description specifically mentions showing tooltips on mouse hover. I have a few follow-up questions.
1. Should the disabled controls be able to receive keyboard focus? Should users be able to Tab over to these buttons?
2. Did the study consider pen or touch? Should tooltips show up on pen hover or touch press-and-hold?
3. The description mentions keyboard shortcuts - can you elaborate? Are these shortcuts the equivalent of pressing the buttons? If so, are the shortcuts still expected to work while the buttons are disabled?

**Comment by jgrieves** [comment]
Should the disabled controls be able to receive keyboard focus? Should users be able to Tab over to these buttons?

That's still a debate for accessibility.  I would follow the traditional Windows pattern of no until the industry or governments settle on this as a requirement.

Did the study consider pen or touch? Should tooltips show up on pen hover or touch press-and-hold?

No the study was only mouse interactions.  Yes to pen hover.  Touch to press and hold is interesting, do we do that for non-disabled controls?  If so, cool and yes.  

The description mentions keyboard shortcuts - can you elaborate? Are these shortcuts the equivalent of pressing the buttons? If so, are the shortcuts still expected to work while the buttons are disabled?

The keyboard shortcuts let the user know how to perform the action of the button when its functional.  Imagine you have a rewind/play/fast forward set of buttons.  Rewind and fast forward have shortcuts of Cntrl + R and Cntrl + F, but are currently disabled.  In that case and in our case the keyboard shortcuts should not work because the functionality is not available.  That would be up to app to properly build (which in the case of our app, Magnifier we do correctly)
